src/linear_algebra/GLA.jl

```diff
@@ -156,6 +159,56 @@ plus_lm(x::AbstractVector) = begin
 end
 zero_lm(n::Int) = x::AbstractVector -> zeros(eltype(x), n)
 
+# LinearOps instance
+#-------------------
+
+@auto_hash_equals struct LinearOpDom
+  N::Int
+end
+
+@instance LinearFunctions(LinearOpDom, LinearOperator) begin
+  @import adjoint, +
+
+  dom(f::LinearOperator) = LinearOpDom(size(f,2))
+  codom(f::LinearOperator) = LinearOpDom(size(f,1))
+
+  compose(f::LinearOperator, g::LinearOperator) = g*f
+  id(V::LinearOpDom) = LOs.opEye(V.N)
+
+  oplus(V::LinearOpDom, W::LinearOpDom) = LinearOpDom(V.N + W.N)
+  oplus(f::LinearOperator, g::LinearOperator) = oplus_lo(f, g)
+  mzero(::Type{LinearOpDom}) = LinearOpDom(0)
+  braid(V::LinearOpDom, W::LinearOpDom) = braid_lo(V.N, W.N)
+  mcopy(V::LinearOpDom) =
+    LOs.opExtension(1:V.N, V.N*2)+LOs.opExtension((V.N+1):(V.N*2), V.N*2)
```

Comment:
I know this is nit-picky, but my brain wants to parse `V.N*2` as `V.N^2` or `V.N**2` (Python style). Maybe write this as `2*V.N`?

Reply:
Oh yeah, I never thought about it, but I agree with that nitpick. Constants before variables just like in hand written math notation.

Reply:
Good call, I'll make those edits. Definitely makes more sense stylistically.